docs/manual/common/guide/devmode/DevEnvironment.md

```diff
@@ -12,8 +12,10 @@ To run all services in Maven, simply run the `lagom:runAll` command:
 $ cd <path to your Lagom project>
 $ mvn lagom:runAll
 INFO ...
-INFO Service hello-impl listening for HTTP on localhost:23966
-INFO Service hello-stream-impl listening for HTTP on localhost:27462
+INFO Service hello-impl listening for HTTP on 127.0.0.1:24266
+INFO Service hello-impl listening for HTTPS on 127.0.0.1:50695
+INFO Service hello-stream-impl listening for HTTP on 127.0.0.1:26230
+INFO Service hello-stream-impl listening for HTTPS on 127.0.0.1:58440
```

Comment:
This is not yet true. This is not being printed, so maybe we should wait until we resolve #1554

Reply:
Oh sorry, I just saw that there is already a PR for it here #1555